--- a/feeds/kenzok8/alist/Makefile
+++ b/feeds/kenzok8/alist/Makefile
@@ -32,6 +32,7 @@ PKG_USE_MIPS16:=0
 PKG_BUILD_FLAGS:=no-mips16
 
 GO_PKG:=github.com/AlistGo/alist
+GO_BUILD_PKG:=github.com/AlistGo/alist/v3/cmd/alist
 GO_PKG_LDFLAGS:= \
 	-X '$(GO_PKG)/v3/internal/conf.BuiltAt=$(shell date '+%Y-%m-%d %H:%M:%S %z')' \
 	-X '$(GO_PKG)/v3/internal/conf.GoVersion=$(shell $(STAGING_DIR_HOSTPKG)/bin/go version | sed 's/go version //')' \
